src/fides/api/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -167,10 +170,10 @@ def create_consent_request(
             "Application redis cache required, but it is currently disabled! Please update your application configuration to enable integration with a redis cache."
         )
 
-    if not data.email and not data.phone_number:
+    if not data.email and not data.phone_number and not data.fides_user_device_id:
         raise HTTPException(
             HTTP_400_BAD_REQUEST,
-            detail="An email address or phone number identity is required",
+            detail="An email address, phone number, or fides_user_device_id is required",
```

Comment:
I haven't tested this yet but I assume if identity verification is on, there would be more we would need to do to bypass this.

Reply:
This isn't intended to be used at all when identity verification is switched on by the server, since a `fides_user_device_id` can't really be used to verify anything. Looks like I'll need to update this block to account for that, thanks!